com.github.rafostar.Clapper.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id": "com.github.rafostar.Clapper",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.38",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.rafostar.Clapper",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-videos",
+        "--talk-name=org.gnome.Shell",
+        "--env=GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0",
+        "--env=GST_VAAPI_ALL_DRIVERS=1"
+    ],
+    "modules": [
+        "lib/glib-networking.json",
```

Comment:
Just to make sure, glib-networking and pango are needed for GTK 4 and gstreamer modules are there because you added some patches?

Reply:
`glib-networking` is an optional dependency of some gstreamer plugins, it handles SSL for URIs. Otherwise app will be unable to play videos from sites with SSL (pretty much any website nowadays).

> pango are needed for GTK 4

Yes.

> gstreamer modules are there because you added some patches

This and because app depends on gstreamer 1.18+. Once 1.18 is available in the runtime I will remove some of them (leave only patched plugins until merged upstream).